### ISSUE TITLE
Números de testes por estado

### ISSUE THREAD

**Issue by whaner** [turicas/covid19-br#147]
Acho que seria interessante termos dados sobre a testagem que está sendo feito no estados já que a relação testagem X confirmados X óbitos é uma informação fundamental para analisar os cenários.
No AC, por exemplo, o boletim traz o total de exames (PCR e rápidos) feitos.
Dúvidas:
- São só os exames feitos pelo poder público?
- É possível obter resultados dos laboratórios particulares?
- Dá para saber quais são os laboratórios aprovados por estado?

**Comment by endersonmaia** [comment]
Alguns estados que estão liberando os microdados (dados detalhados por caso/paciente) já informam o tipo do teste.

Tirando AL como exemplo, ele mostra o tipo do teste nos microdados, e no boletim em PDF ele mostra um panorama da quantidade de testes disponíveis.

Para estados com dados menos detalhados, daria pra inferir o total de testes considerando que todo caso confirmado foi feito o teste e somar ao total de casos descartados.

Daí, adicionar duas colunas na planilha `testados` que contabilizaria o total de testes até aquela data.

ex.:

```csv
municipio, confirmados, obitos, testados
TOTAL NO ESTADO, 1000, 100, 1500
Cidade X, 100, 10, 150
...
```

**Comment by ecrisol** [comment]
Vários estados já estão disponibilizando em paínes os dados: DF - https://salasit.saude.df.gov.br/publicos-leitos-covid-internacao/; RS - https://covid.saude.rs.gov.br/
MG - https://www.saude.mg.gov.br/coronavirus/painel
ES - https://coronavirus.es.gov.br/leitos-uti
GO - https://www.saude.go.gov.br/coronavirus
RO - http://covid19.sesau.ro.gov.br/Home/Leitos

**Comment by ecrisol** [comment]
Em http://plataforma.saude.gov.br/coronavirus/virus-respiratorios/ tem a situação nacional dos exames.

**Comment by turicas** [comment]
Alguns links que podem ajudar:

- https://repositoriodatasharingfapesp.uspdigital.usp.br/
- https://ourworldindata.org/grapher/positive-rate-daily-smoothed
- https://coronavirusbra1.github.io/

**Comment by luizcelsojr** [comment]
Sim, esse assunto é muito importe. É bem triste não termos esses dados que são divulgados por diversos outros países.

Eu tenho trabalhado nisso nos dias, usando dados liberados pela Fapesp (link do comentário do @turicas ). Em breve vou usar também os dados do e-SUS-VE. Mas ontem tivemos um desenvolvimento interessante:

[Laboratórios terão 24 horas para enviar dados de testes aplicados ao Ministério da Saúde](https://www1.folha.uol.com.br/equilibrioesaude/2020/07/laboratorios-terao-24-horas-para-enviar-dados-de-testes-aplicados-ao-ministerio-da-saude.shtml)

Então o ideal é acompanhar esse processo e ver se o governo vai liberar esse dado.

Se não liberarem, acho bacana a gente se envolver e tentar obter a partir dos boletins. Infelizmente a divulgação pelas secretarias não é uniforme, então teríamos alguns problemas nesse caminho.

De qualquer forma, a taxa de positividade é muito importante, sobretudo neste momento.

Quem quiser mais detalhes, este relatório é excelente:

[Boletim da "Rede de Pesquisa Solidária" sobre taxas de testagem positiva no Brasil](https://redepesquisasolidaria.org/wp-content/uploads/2020/06/boletimpps_13_26junho.pdf
